GNUmakefile

```diff
@@ -23,7 +23,14 @@ test:
 	echo $(TEST) | \
 		xargs -t ${GOCMD} test -v $(TESTARGS) -timeout=10m
 
-testacc:
+testaccmoved:
+	# go through all the directories in internal/resources and run tests
+	for d in $$(find ./internal/resources -maxdepth 2 -mindepth 2 -type d); do \
```

Comment:
These packages should already be covered by `$(TEST)`. The for loop over directories in internal/ doesn't seem necessary and I suspect we are retesting packages.